benchmarks/README.md

```diff
@@ -64,10 +68,14 @@ _________________
   or assigned to the [master node](https://cloud.google.com/compute/docs/connect/add-ssh-keys#after-vm-creation) only.
 - Ideally, all GCP components used in benchmark should be in the same location (Storage bucket, Dataproc Metastore service and Dataproc cluster).
 
-#### Prepare GCS bucket
+There are two ways to create infrastructure required for benchmarks - using provided [Terraform template](infrastructure/gcp/terraform/README.md) or manually.
+
+#### Create infrastructure manually
```

Comment:
Since using terraform is not separation section, I dont think this needs to be a section on its own.

Reply:
Ok, I removed the header (both in line 26 and 73).